src/util.c

```diff
@@ -559,6 +559,17 @@ cc_oci_enable_networking (void)
 	return enable;
 }
 
+/*!
+ * Convert the value stored in buffer to little endian
+ *
+ * \param buf Buffer storing the big endian value
+ *
+ * \return \c guint32 in big endian order.
+ */
+guint32 cc_oci_get_big_endian_32(const char *buf) {
+	return (guint32)(buf[0] >> 24 | buf[1] >> 16 | buf[2] >> 8 | buf[3]);
```

Comment:
@gorozco1  @jcvenegas This needs to as
```
uint32_t guint32 cc_oci_get_big_endian_32(uint8_t *buf){
        return (uint32_t)(buf[0] << 24 | buf[1] << 16 | buf[2] << 8 | buf[3]);
}
```
Note the use of unsigned pointer in the parameter as well, required for correct bit operations. I had made a similar change in shim as well, but got lost in the many rebases I did with 2 different branches since then. Going to be pushing that in my next PR for shim.